chroma-manager/chroma-manager.spec

```diff
@@ -297,6 +285,7 @@ echo "run \"chroma-config setup\""
 service chroma-supervisor stop
 # remove the /static/ dir of files that was created by Django's collectstatic
 rm -rf %{manager_root}/static
+-find /usr/share/chroma-manager/ -name "*.pyc" -exec rm -f {} \;
```

Comment:
This seems an odd pattern.  I can't imagine every python module RPM has to do this.

I'm pretty sure that at RPM build time, the `.pyc`s are created and thus are part of the RPMs inventory an should therefor be removed by `rpm` when the RPM is removed.